tools/c7n_org/README.md

```diff
@@ -42,8 +45,29 @@ accounts:
 ...
 ```
 
-We also distribute a script `orgaccounts.py` that can generate this config file
-from the AWS Organizations API.
+Example Azure Config File:
+
+```yaml
+subscriptions:
+- name: Subscription-1
+  subscription_id: a1b2c3d4-e5f6-g7h8i9...
+- name: Subscription-2
+  subscription_id: 1z2y3x4w-5v6u-7t8s9r...
+```
+
+### Config File Generation
+
+We also distribute scripts to generate the necessary config file.
+
+- For **AWS**, the script `orgaccounts.py` generates a config file from the AWS Organizations API
+- For **Azure**, the script `azuresubs.py` generates a config file from the Azure Resource Management API
+    - Please see the **Additional Azure Instructions** section at the bottom of the page for initial setup
+    
+```shell
+python orgaccounts.py -f output.yml
+OR
```

Comment:
minor: separate the two examples into separate code blocks in this format and move it below line 62 and 63:

```shell
$ python orgaccounts.py -f output.yml
```

```shell
$ python azuresubs.py -f output.yml
```